deploy/chaos_crds.yaml

```diff
@@ -149,7 +149,9 @@ spec:
                                   minLength: 1
                                 insecureSkipVerify:
                                   type: boolean
-                                  minLength: 1 
+                                  minLength: 1
```

Comment:
Do boolean support minLength field? Can we verify once?

Reply:
I have run it without ```minLength```.
Working fine,